flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -196,11 +196,22 @@ under the License.
 				</exclusion>
 			</exclusions>
 		</dependency>
+
+		<!-- Table ecosystem -->
+		<!-- Projects depending on this project, won't depend on flink-table-*. -->
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-table-api-java_${scala.binary.version}</artifactId>
```

Comment:
What is this dependency needed for? table-planner is required for `AppendStreamTableSink`.